### ISSUE TITLE
Version selector is confusing

### ISSUE THREAD

**Issue by meijeru** [red/docs#14]
Version 0.6.0 does not have a version selector, but 0.6.1 does, and it allows selection of 0.6.0, 0.6.1 and 0.6.2. However, 0.6.2 is not (yet) available. Moreover, the default value shown in the version selector is 0.6.2, so the impression is created that this version is selectable. I propose that in each version, the version selector shows THAT version as default. This is a better way of finding out which version one is in, than to look at the URL in the address box.

**Comment by dockimbel** [comment]
We have no control over that feature, we can just enable or disable version selection. The current selected version is shown in the URL.

**Comment by meijeru** [comment]
OK but a pity it is.

**Comment by 9214** [comment]
As of now, `v0.6.3` selector leads to [nowhere](https://doc.red-lang.org/v/v0.6.3/).

**Comment by greggirwin** [comment]
Yes, I believe this was a branch/tag issue, which was never resolved. @x8x do you recall?

**Comment by x8x** [comment]
Yes, it was tagged wrongly, I requested access to the gitbook account as it was impossible to set up a new account on legacy for test/fix it but didn't get it.
We need to move away from gitbook, it is legacy and has had enough issues this year.
I'll have something ready for comments in a few days. Great that Lucinda has eyes on that too!
